### PR TITLE
fix #918

### LINKED ISSUE #918: ESP8266 build broken in current master
Just pulled current master and esp8266 build fails
```
GEN build/genhdr/qstrdefs.generated.h
FREEZE modules
build/shared-bindings/pulseio/PulseIn.o:(.text.pulseio_pulsein_obj_get_paused+0x0): undefined reference to `common_hal_pulseio_pulsein_get_paused'
build/shared-bindings/pulseio/PulseIn.o: In function `pulseio_pulsein_obj_get_paused':
PulseIn.c:(.text.pulseio_pulsein_obj_get_paused+0x18): undefined reference to `common_hal_pulseio_pulsein_get_paused'
Makefile:250: recipe for target 'build/firmware.elf' failed
make: *** [build/firmware.elf] Error 1
```

### PR DESCRIPTION
**Pull request by jerryneedell**
Add stub for common_hal_pulseio_pulsein_get_paused  to make ESP8266 build compatible with PR #901

### CODE REVIEW

File: ports/esp8266/common-hal/pulseio/PulseIn.c [hunk 1]
```diff
@@ -62,6 +62,10 @@ uint16_t common_hal_pulseio_pulsein_get_maxlen(pulseio_pulsein_obj_t* self) {
     return 0;
 }
 
+bool common_hal_pulseio_pulsein_get_paused(pulseio_pulsein_obj_t* self) {
+    return true;
```
Comment (dhalbert) [inline ports/esp8266/common-hal/pulseio/PulseIn.c]: Since pause() does nothing, let's return `false` instead.

**Comment by jerryneedell** [comment]
Should this change also be made for nrf?

**Comment by jerryneedell** [comment]
I just copied it from the nrf portion of #901

**Comment by dhalbert** [comment]
Sure. It's all temporary code anyway, but at least let's make the semantics approximately right. Thanks.

**Pull request merged**